hpe3parclient/client.py

```diff
@@ -1287,7 +1287,7 @@ def createSnapshot(self, name, copyOfName, optional=None):
     # Host Set methods
     def findHostSet(self, name):
         """
-        Find the Host Set name for a host.
+	Find the Host Set name for a host.
```

Comment:
Why this change in indentation, please could you undo all changes that are unrelated to your commit?